clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/KubernetesV2ProviderSynchronizableSpec.groovy

```diff
@@ -32,28 +32,29 @@ import com.netflix.spinnaker.clouddriver.security.AccountCredentials
 import com.netflix.spinnaker.clouddriver.security.AccountCredentialsRepository
 import com.netflix.spinnaker.clouddriver.security.ProviderVersion
 import com.netflix.spinnaker.kork.configserver.ConfigFileService
+import groovy.transform.CompileStatic
```

Comment:
Should we start adding this to any groovy tests we touch?

Reply:
It's not always trivial to add, as there are some Groovy constructs that aren't allowed in `@CompileStatic`.  (`@TypeChecked` is an alternative in that case that will still throw type errors if you have them.)

I've actually had some trouble in the past getting `@CompileStatic` to work with tests because I think some of the `Spock` constructs we use don't work well with it.  But in cases where it is easy to add it to the class I'd definitely say go ahead.

I added this because I was getting hard-to-debug errors in the tests and wanted some type-safety to help me out.